src/backend/tests/test-data/users.test-data.ts

```diff
@@ -19,3 +19,13 @@ export const superman = {
   role: Role.ADMIN,
   googleAuthId: 's'
 };
+
+export const guestUser = {
```

Comment:
if making a new user, give them unique info than the old ones because then we can distinguish them better

Reply:
feel free to do any character you want, i was doing superhero stuff earlier. i think it makes it more fun lol